public/index.html

```diff
@@ -26,8 +26,18 @@
         </div>
         <div class="logo"><img src="asset/logo.png"></div>
         <!-- <button id="toggle-theme">☀️</button> -->
+         
     </header>
     <div class="content">
+        <div class="section">
```

Comment:
do not add the about here, create a separate about page

Reply:
Now I have added separate about page.